src/PrestaShopBundle/Resources/views/Admin/Layout/light_layout.html.twig

```diff
@@ -39,8 +39,16 @@
   {% endblock %}
 </head>
 
+{# Construct CSS classes based on version numbers #}
```

Comment:
Having this code in three files proves the point of having to separate it somewhere upper 😅